src/main/resources/locales/cy/translation/common.json

```diff
@@ -4,7 +4,7 @@
   "continue": "Parhau",
   "view": "Gweld",
   "change": "Newid",
-  "cancel": "Welsh translation required",
+  "cancel": "Allgofnodi",
```

Comment:
Don't think this is correct. This means 'Sign out'

Reply:
Fixed